hasher/src/roinput.rs

```diff
@@ -327,14 +327,10 @@ mod tests {
             Fq::from_hex("18b7ef420128e69623c0c0dcfa28d47a029d462720deb769d7b5dd6f17444216")
                 .expect("failed to create scalar");
         let roi = ROInput::new().append_scalar(scalar);
-        assert_eq!(roi.bits.len(), 255);
+        assert_eq!(roi.fields.len(), 1);
```

Comment:
As mentioned earlier, it would be better to not change the actual behavior of append_scalar, and add a different method instead. Both methods should be documented.

Reply:
As a consequence, these tests should not be updated.